scripts/west_commands/runners/dfu.py

```diff
@@ -42,13 +46,11 @@ def name(cls):
 
     @classmethod
     def capabilities(cls):
-        return RunnerCaps(commands={'flash'}, flash_addr=True)
+        return RunnerCaps(commands={'flash'}, dev_id=True, flash_addr=True)
 
     @classmethod
     def do_add_parser(cls, parser):
         # Required:
-        parser.add_argument("--pid", required=True,
-                            help="USB VID:PID of the board")
```

Comment:
For compatibility, please change to: `parser.add_argument('--pid', dest='dev_id')`, removing `required=True` and changing the destination attribute in the final args object to match --dev-id's. (you can change `help` to mention the consolidation if you think that'd be good here as well.)